backend-tests/tests/test_inventory.py

```diff
@@ -321,3 +293,278 @@ def test_fail_no_attr_value(self, user):
                 "PATCH", inventory.URL_DEVICE_ATTRIBUTES, payload
             )
             assert r.status_code == 400
+
+
+class TestDeviceFilteringEnterprise:
+    @property
+    def logger(self):
+        try:
+            return self._logger
+        except AttributeError:
+            self._logger = logging.getLogger(self.__class__.__name__)
+        return self._logger
+
+    def dict_to_inventoryattrs(self, d, scope=None):
+        attr_list = []
+        for key, value in d.items():
+            attr = {"name": key, "value": value}
+            if scope is not None:
+                attr["scope"] = scope
+            attr_list.append(attr)
+
+        return attr_list
+
+    def test_search_v2(self, tenants_users):
+        NUM_DEVICES = 100
+
+        useradmm = ApiClient(useradm.URL_MGMT)
+        devauthd = ApiClient(deviceauth.URL_DEVICES)
+        devauthm = ApiClient(deviceauth.URL_MGMT)
+        invd = ApiClient(inventory.URL_DEV)
+        invm_v2 = ApiClient(inventory_v2.URL_MGMT)
+
+        # Initialize devices and API tokens
+        tenants = [t for t in tenants_users]
```

Comment:
Unnecessary, we can use `tenant_users`, right?

Reply:
yes, fixed